### DISCUSSION TITLE
Is it possible to center the selected item?

### DISCUSSION THREAD

**Discussion by mircobarachetti** [asmyshlyaev177/react-horizontal-scrolling-menu#140] [General]
Hello ,

Is it possible to center the selected item?

**Comment by mircobarachetti** [comment]
@asmyshlyaev177

**Comment by asmyshlyaev177** [comment]
Not sure, working on next version now https://github.com/asmyshlyaev177/react-horizontal-scrolling-menu/issues/137

**Reply by mircobarachetti** [reply]
Thank you. How can use the beta4 in my code ? When I tried to add the code into my repository there is a problem with ScrollMenu (doesn't exist constructor). With the 0.7.10 version no problem.

**Reply by asmyshlyaev177** [reply]
Can add like `yarn add react-horizontal-scrolling-menu@beta`, but api is different and docs aren't ready yeat, can look example.

**Reply by mircobarachetti** [reply]
But with the beta versione I 've this error:
![image](https://user-images.githubusercontent.com/9607467/123288470-e483ca00-d50f-11eb-91bb-e3206158d3ac.png)
The code is the same. Is there an error in the beta version?

**Reply by asmyshlyaev177** [reply]
Check how you import it.
Should be like this `import { ScrollMenu, VisibilityContext, api } from "react-horizontal-scrolling-menu";`

Related SO https://stackoverflow.com/questions/34971155/typescript-error-ts2604-how-to-fix
